docker-compose.yml

```
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  app:
+    build:
+      context: .
+      args:
+        - VERSION=$MINECRAFT_VERSION
+    environment:
+      EULA: $EULA # TRUE or FALSE
+    volumes:
+      - $SPIGOT_DIR:/opt/spigot
+    ports:
+      - "25565:25565/tcp"
+      - "25565:25565/udp"
```

```suggestion
      - "25575:25575/udp"
```

Minecraft のゲームクライアントとの通信では UDP は使いません。
RCON を使う場合は UDP を使いますが、その場合はデフォルトで 25575/udp です。

---

ありがとうございます。知らなかった。
RCON のために 25575/udp は追加します。

ただ、25565/udp については、調べてみると query(GameSpy4 protocol) で使うみたいなので、残しておきたいと考えています。

---

GameSpy4は知らなかった……
了解です

---

cf9ae0a で変更されたことを確認しました :+1: